swim.go

```diff
@@ -182,7 +214,14 @@ func (s *SWIM) handle(msg pb.Message) {
 
 	switch msg.Payload.(type) {
 	case *pb.Message_Ping:
-		// handle ping
+		s.messageEndpoint.Send(msg.Address, pb.Message{
+			Seq: "",
+			Address: "",
+			Payload: &pb.Message_Ack{
+				Ack: &pb.Ack{Payload: ""},
+			},
+			PiggyBack: &pb.PiggyBack{},
+		})
```

Comment:
get piggyback from pbkstore if it is not empty

Reply:
pbkstore push and get []byte type. I don't understand why pbkstore use []byte type.  Does PiggyBack convey member information(ex> address, seq(id), incarnation number)?